.github/workflows/e2e-leg-4.yml

```diff
@@ -72,6 +77,10 @@ jobs:
         export E2E_TEST_DIRS="tests/e2e-leg-4"
         export VERTICA_DEPLOYMENT_METHOD=${{ inputs.vertica-deployment-method }}
         if [ "${VERTICA_DEPLOYMENT_METHOD}" != "vclusterops" ]; then E2E_TEST_DIRS+=" tests/e2e-leg-4-at-only"; fi
+        export VERTICA_SUPERUSER_NAME=${{ inputs.vertica-superuser-name }}
```

Comment:
I don't think we are setting this correctly. When I look at your CI run I see dbadmin being used:
```
Vertica server image name: docker.io/vertica/vertica-k8s-private:latest-minimal-master
Base vertica server image name for upgrade tests: <not-set>
Vertica logger image name: vertica-logger:kind
Endpoint: https://minio.kuttl-e2e-s3/
Protocol: s3://
Communal Path Prefix: s3://nimbusdb/
Using private registry: NO
Add server mounts: NO
Deployment method: vclusterops
Vertica superuser name: dbadmin
```
https://github.com/vertica/vertica-kubernetes/actions/runs/6785933635/job/18448129113

The new parameter was added to `workflow_dispatch`. That gets called when you trigger just this workflow. There are a set of parameters for `workflow_call`. This is used when another workflow calls this workflow, which is what happens during a PR. Can you add the new parameter to `workflow_call`?